WORKSPACE

```diff
@@ -0,0 +1,9 @@
+workspace(name="example")
+
+load("@example//:defs.bzl", "read_bazel_out")
+read_bazel_out(
+    name = "test_rule",
+    typescript_out = "typescript_out",
+    rebuild = "@example//:.bazelexec.stamp",
```

Comment:
Note: we won't need this rebuild thing because the rule will depend on all of the typescript files on disk, so changes will properly re-trigger typescript running.